src/assets/styles/_global.scss

```diff
@@ -3,6 +3,12 @@
 @import 'tailwindcss/components';
 @import './components/_button.scss';
 
+@import 'tailwindcss/utilities';
+@import 'tailwindcss/base';
+
+@import 'tailwindcss/components';
+@import './components/_button.scss';
+
 @import 'tailwindcss/utilities';
```

Comment:
⚠️ Please don't re-arrange the order of imports.

If you checked the tailwind docs, they specifically say that when working with pre-processors like scss, the order of imports is crucial, especially if you have custom class components like that `_button.scss`.

My suggestion is you don't modify this file, and leave it as it was.

Unless there is something you are adding. But from the looks of things, you only copied the utilities import, and pasted it beneath, and added nothing new.